### ISSUE TITLE
Bug: Kitchen Owl Reverse Proxy Not working on Android App

### ISSUE THREAD

**Issue by DevilsDesigns** [TomBursch/kitchenowl#513]
### Is this urgent?

No

### What parts are affected

Both

### What is the server version

0.5.2

### What is the client version

0.6.0

### What platform are you using

Android

### What's the problem 🤔

i get an error the server version is not supported, please update the backend. Im using Unraids KitchenOwl All in One. Its as far as i know the latest update. 
![image](https://github.com/user-attachments/assets/356d58d5-b7cc-4fdd-a78b-bf8e6df24494)
![image](https://github.com/user-attachments/assets/f88eec17-8847-4aa7-8fa9-fd6275ed26ae)
It also loads just fine on browser.
ive tried clearing cache and data as well on the app on android. still wont connect

### Share your logs

```Shell
[pid: 30|app: 0|req: 145/171] ::ffff:192.168.0.111 () {56 vars in 1351 bytes} [Fri Sep 20 21:13:28 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bmwzvv&sid=sVo09xUdZ9SEyY56AAAA => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 198)
[pid: 30|app: 0|req: 146/172] ::ffff:192.168.0.111 () {50 vars in 1243 bytes} [Fri Sep 20 21:13:28 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bmwzvv&sid=sVo09xUdZ9SEyY56AAAA => generated 1 bytes in 25007 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 148/173] ::ffff:192.168.0.111 () {56 vars in 1351 bytes} [Fri Sep 20 21:13:53 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bmxj74&sid=sVo09xUdZ9SEyY56AAAA => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 198)
[pid: 30|app: 0|req: 149/174]  () {16 vars in 245 bytes} [Fri Sep 20 21:14:03 2024] GET /api/health/8M4F88S8ooi4sMbLBfkkV7ctWwgibW6V => generated 71 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 198)
[pid: 30|app: 0|req: 149/175] ::ffff:192.168.0.111 () {50 vars in 1243 bytes} [Fri Sep 20 21:13:53 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bmxj74&sid=sVo09xUdZ9SEyY56AAAA => generated 1 bytes in 25012 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 150/176] ::ffff:192.168.0.111 () {56 vars in 1350 bytes} [Fri Sep 20 21:14:18 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bmy2ij&sid=sVo09xUdZ9SEyY56AAAA => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 151/177] ::ffff:192.168.0.111 () {50 vars in 1242 bytes} [Fri Sep 20 21:14:18 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bmy2ij&sid=sVo09xUdZ9SEyY56AAAA => generated 1 bytes in 25017 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 198)
[pid: 30|app: 0|req: 153/178] ::ffff:192.168.0.111 () {56 vars in 1351 bytes} [Fri Sep 20 21:14:43 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bmylu2&sid=sVo09xUdZ9SEyY56AAAA => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 154/179]  () {16 vars in 245 bytes} [Fri Sep 20 21:15:04 2024] GET /api/health/8M4F88S8ooi4sMbLBfkkV7ctWwgibW6V => generated 71 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 154/180] ::ffff:192.168.0.111 () {50 vars in 1243 bytes} [Fri Sep 20 21:14:43 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bmylu2&sid=sVo09xUdZ9SEyY56AAAA => generated 1 bytes in 25002 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 198)
[pid: 30|app: 0|req: 156/181] ::ffff:192.168.0.111 () {56 vars in 1351 bytes} [Fri Sep 20 21:15:08 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bmz556&sid=sVo09xUdZ9SEyY56AAAA => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 156/182] ::ffff:192.168.0.111 () {50 vars in 1243 bytes} [Fri Sep 20 21:15:08 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bmz556&sid=sVo09xUdZ9SEyY56AAAA => generated 1 bytes in 25007 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 198)
[pid: 30|app: 0|req: 157/183] ::ffff:192.168.0.111 () {56 vars in 1351 bytes} [Fri Sep 20 21:15:33 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bmzogg&sid=sVo09xUdZ9SEyY56AAAA => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 198)
[pid: 30|app: 0|req: 158/184] ::ffff:192.168.0.111 () {50 vars in 1243 bytes} [Fri Sep 20 21:15:33 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bmzogg&sid=sVo09xUdZ9SEyY56AAAA => generated 1 bytes in 25011 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 159/185] ::ffff:192.168.0.111 () {56 vars in 1351 bytes} [Fri Sep 20 21:15:58 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn07rs&sid=sVo09xUdZ9SEyY56AAAA => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 161/186]  () {16 vars in 245 bytes} [Fri Sep 20 21:16:04 2024] GET /api/health/8M4F88S8ooi4sMbLBfkkV7ctWwgibW6V => generated 71 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 199)
[pid: 30|app: -1|req: -1/187] ::ffff:192.168.0.111 () {52 vars in 899 bytes} [Fri Sep 20 21:16:05 2024] GET /household => generated 2804 bytes in 0 msecs via sendfile() (HTTP/1.1 200) 2 headers in 87 bytes (1 switches on core 199)
[pid: 30|app: -1|req: -1/188] ::ffff:192.168.0.111 () {48 vars in 826 bytes} [Fri Sep 20 21:16:06 2024] GET /version.json?cachebuster=1726892169230 => generated 91 bytes in 0 msecs via sendfile() (HTTP/1.1 200) 2 headers in 85 bytes (1 switches on core 199)
[pid: 30|app: 0|req: 162/189] ::ffff:192.168.0.111 () {48 vars in 837 bytes} [Fri Sep 20 21:16:06 2024] GET /api/health/8M4F88S8ooi4sMbLBfkkV7ctWwgibW6V => generated 71 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 163/190] ::ffff:192.168.0.111 () {50 vars in 1127 bytes} [Fri Sep 20 21:16:06 2024] GET /api/auth/refresh => generated 822 bytes in 370 msecs (HTTP/1.1 200) 2 headers in 72 bytes (3 switches on core 199)
[pid: 30|app: -1|req: -1/191] ::ffff:192.168.0.111 () {52 vars in 862 bytes} [Fri Sep 20 21:16:06 2024] GET /flutter_service_worker.js?v=1278757089 => generated 9705 bytes in 0 msecs via sendfile() (HTTP/1.1 200) 2 headers in 87 bytes (1 switches on core 199)
[pid: 30|app: 0|req: 164/192] ::ffff:192.168.0.111 () {50 vars in 1183 bytes} [Fri Sep 20 21:16:06 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn0e3i => generated 97 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 165/193] ::ffff:192.168.0.111 () {50 vars in 1110 bytes} [Fri Sep 20 21:16:06 2024] GET /api/user => generated 1044 bytes in 80 msecs (HTTP/1.1 200) 2 headers in 73 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 167/194] ::ffff:192.168.0.111 () {56 vars in 1341 bytes} [Fri Sep 20 21:16:06 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn0e5r&sid=DnxwNUvcmU0NmvjhAAAE => generated 2 bytes in 248 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 167/195] ::ffff:192.168.0.111 () {50 vars in 1233 bytes} [Fri Sep 20 21:16:06 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn0e5r&sid=DnxwNUvcmU0NmvjhAAAE => generated 32 bytes in 248 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 168/196] ::ffff:192.168.0.111 () {50 vars in 1121 bytes} [Fri Sep 20 21:16:07 2024] GET /api/household => generated 685 bytes in 82 msecs (HTTP/1.1 200) 2 headers in 72 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 169/197] ::ffff:192.168.0.111 () {50 vars in 1243 bytes} [Fri Sep 20 21:15:58 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn07rs&sid=sVo09xUdZ9SEyY56AAAA => generated 1 bytes in 25007 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 198)
[pid: 30|app: 0|req: 170/198] ::ffff:192.168.0.111 () {56 vars in 1351 bytes} [Fri Sep 20 21:16:23 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn0r31&sid=sVo09xUdZ9SEyY56AAAA => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 198)
[pid: 30|app: 0|req: 171/199] ::ffff:192.168.0.111 () {50 vars in 1243 bytes} [Fri Sep 20 21:16:07 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn0eet&sid=DnxwNUvcmU0NmvjhAAAE => generated 1 bytes in 24587 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 171/200] ::ffff:192.168.0.111 () {50 vars in 1243 bytes} [Fri Sep 20 21:16:23 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn0r32&sid=sVo09xUdZ9SEyY56AAAA => generated 1 bytes in 25001 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 172/201] ::ffff:192.168.0.111 () {56 vars in 1351 bytes} [Fri Sep 20 21:16:48 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn1ae5&sid=sVo09xUdZ9SEyY56AAAA => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 197)
[pid: 30|app: -1|req: -1/202] ::ffff:192.168.0.111 () {48 vars in 818 bytes} [Fri Sep 20 21:16:54 2024] GET /version.json?cachebuster=1726892217261 => generated 91 bytes in 0 msecs via sendfile() (HTTP/1.1 200) 2 headers in 85 bytes (1 switches on core 197)
[pid: 30|app: 0|req: 174/203] ::ffff:192.168.0.111 () {48 vars in 829 bytes} [Fri Sep 20 21:16:54 2024] GET /api/health/8M4F88S8ooi4sMbLBfkkV7ctWwgibW6V => generated 71 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 175/204] ::ffff:192.168.0.111 () {50 vars in 1128 bytes} [Fri Sep 20 21:16:54 2024] GET /api/auth/refresh => generated 822 bytes in 340 msecs (HTTP/1.1 200) 2 headers in 72 bytes (3 switches on core 197)
[pid: 30|app: -1|req: -1/205] ::ffff:192.168.0.111 () {52 vars in 863 bytes} [Fri Sep 20 21:16:54 2024] GET /flutter_service_worker.js?v=1278757089 => generated 9705 bytes in 0 msecs via sendfile() (HTTP/1.1 200) 2 headers in 87 bytes (1 switches on core 197)
[pid: 30|app: 0|req: 176/206] ::ffff:192.168.0.111 () {50 vars in 1184 bytes} [Fri Sep 20 21:16:54 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn1f42 => generated 97 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 177/207] ::ffff:192.168.0.111 () {50 vars in 1111 bytes} [Fri Sep 20 21:16:54 2024] GET /api/user => generated 1044 bytes in 77 msecs (HTTP/1.1 200) 2 headers in 73 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 178/208] ::ffff:192.168.0.111 () {56 vars in 1342 bytes} [Fri Sep 20 21:16:54 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn1f58&sid=lK1_PiI5TIlPX5J8AAAG => generated 2 bytes in 168 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 179/209] ::ffff:192.168.0.111 () {50 vars in 1234 bytes} [Fri Sep 20 21:16:54 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn1f59&sid=lK1_PiI5TIlPX5J8AAAG => generated 32 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 198)
[pid: 30|app: 0|req: 180/210] ::ffff:192.168.0.111 () {50 vars in 1121 bytes} [Fri Sep 20 21:16:54 2024] GET /api/household => generated 685 bytes in 83 msecs (HTTP/1.1 200) 2 headers in 72 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 182/211]  () {16 vars in 245 bytes} [Fri Sep 20 21:17:04 2024] GET /api/health/8M4F88S8ooi4sMbLBfkkV7ctWwgibW6V => generated 71 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 198)
[pid: 30|app: 0|req: 182/212] ::ffff:192.168.0.111 () {50 vars in 1243 bytes} [Fri Sep 20 21:16:48 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn1ae5&sid=sVo09xUdZ9SEyY56AAAA => generated 1 bytes in 25001 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 182/213] ::ffff:192.168.0.111 () {50 vars in 1243 bytes} [Fri Sep 20 21:16:55 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn1fc2&sid=lK1_PiI5TIlPX5J8AAAG => generated 1 bytes in 24671 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 183/214]  () {16 vars in 245 bytes} [Fri Sep 20 21:18:04 2024] GET /api/health/8M4F88S8ooi4sMbLBfkkV7ctWwgibW6V => generated 71 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 197)
[pid: 30|app: -1|req: -1/215] ::ffff:192.168.0.111 () {48 vars in 818 bytes} [Fri Sep 20 21:18:14 2024] GET /version.json?cachebuster=1726892297347 => generated 91 bytes in 0 msecs via sendfile() (HTTP/1.1 200) 2 headers in 85 bytes (1 switches on core 197)
[pid: 30|app: 0|req: 184/216] ::ffff:192.168.0.111 () {48 vars in 829 bytes} [Fri Sep 20 21:18:14 2024] GET /api/health/8M4F88S8ooi4sMbLBfkkV7ctWwgibW6V => generated 71 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 185/217] ::ffff:192.168.0.111 () {50 vars in 1128 bytes} [Fri Sep 20 21:18:14 2024] GET /api/auth/refresh => generated 822 bytes in 352 msecs (HTTP/1.1 200) 2 headers in 72 bytes (3 switches on core 197)
[pid: 30|app: -1|req: -1/218] ::ffff:192.168.0.111 () {52 vars in 863 bytes} [Fri Sep 20 21:18:14 2024] GET /flutter_service_worker.js?v=1278757089 => generated 9705 bytes in 0 msecs via sendfile() (HTTP/1.1 200) 2 headers in 87 bytes (1 switches on core 197)
[pid: 30|app: 0|req: 186/219] ::ffff:192.168.0.111 () {50 vars in 1184 bytes} [Fri Sep 20 21:18:14 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn34wo => generated 97 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 187/220] ::ffff:192.168.0.111 () {50 vars in 1111 bytes} [Fri Sep 20 21:18:14 2024] GET /api/user => generated 1044 bytes in 76 msecs (HTTP/1.1 200) 2 headers in 73 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 188/221] ::ffff:192.168.0.111 () {56 vars in 1342 bytes} [Fri Sep 20 21:18:14 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn34xx&sid=BCvXz_PTBp7yLoVYAAAI => generated 2 bytes in 178 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 189/222] ::ffff:192.168.0.111 () {50 vars in 1234 bytes} [Fri Sep 20 21:18:15 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn34xx&sid=BCvXz_PTBp7yLoVYAAAI => generated 32 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 190/223] ::ffff:192.168.0.111 () {50 vars in 1121 bytes} [Fri Sep 20 21:18:15 2024] GET /api/household => generated 685 bytes in 82 msecs (HTTP/1.1 200) 2 headers in 72 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 192/224] ::ffff:192.168.0.111 () {50 vars in 1134 bytes} [Fri Sep 20 21:18:16 2024] GET /api/household/1 => generated 683 bytes in 97 msecs (HTTP/1.1 200) 2 headers in 72 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 193/225] ::ffff:192.168.0.111 () {50 vars in 1160 bytes} [Fri Sep 20 21:18:16 2024] GET /api/household/1/shoppinglist => generated 99 bytes in 77 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 194/226] ::ffff:192.168.0.111 () {50 vars in 1150 bytes} [Fri Sep 20 21:18:16 2024] GET /api/household/1/planner => generated 3 bytes in 81 msecs (HTTP/1.1 200) 2 headers in 70 bytes (3 switches on core 198)
[pid: 30|app: 0|req: 195/227] ::ffff:192.168.0.111 () {50 vars in 1180 bytes} [Fri Sep 20 21:18:16 2024] GET /api/household/1/planner/recent-recipes => generated 3 bytes in 101 msecs (HTTP/1.1 200) 2 headers in 70 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 196/228] ::ffff:192.168.0.111 () {50 vars in 1180 bytes} [Fri Sep 20 21:18:16 2024] GET /api/household/1/expense/categories => generated 3 bytes in 79 msecs (HTTP/1.1 200) 2 headers in 70 bytes (3 switches on core 198)
[pid: 30|app: 0|req: 197/229] ::ffff:192.168.0.111 () {50 vars in 1186 bytes} [Fri Sep 20 21:18:16 2024] GET /api/household/1/planner/suggested-recipes => generated 3 bytes in 102 msecs (HTTP/1.1 200) 2 headers in 70 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 198/230] ::ffff:192.168.0.111 () {50 vars in 1171 bytes} [Fri Sep 20 21:18:16 2024] GET /api/household/1/expense?view=0 => generated 3 bytes in 78 msecs (HTTP/1.1 200) 2 headers in 70 bytes (3 switches on core 198)
[pid: 30|app: 0|req: 199/231] ::ffff:192.168.0.111 () {50 vars in 1179 bytes} [Fri Sep 20 21:18:17 2024] GET /api/shoppinglist/1/items?orderby=0 => generated 3302 bytes in 104 msecs (HTTP/1.1 200) 2 headers in 73 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 200/232] ::ffff:192.168.0.111 () {50 vars in 1160 bytes} [Fri Sep 20 21:18:17 2024] GET /api/household/1/category => generated 1820 bytes in 96 msecs (HTTP/1.1 200) 2 headers in 73 bytes (3 switches on core 198)
[pid: 30|app: 0|req: 201/233] ::ffff:192.168.0.111 () {50 vars in 1189 bytes} [Fri Sep 20 21:18:17 2024] GET /api/shoppinglist/1/recent-items?limit=9 => generated 3 bytes in 93 msecs (HTTP/1.1 200) 2 headers in 70 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 201/234] ::ffff:192.168.0.111 () {50 vars in 1243 bytes} [Fri Sep 20 21:18:15 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn354x&sid=BCvXz_PTBp7yLoVYAAAI => generated 1 bytes in 24672 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 203/235] ::ffff:192.168.0.111 () {56 vars in 1359 bytes} [Fri Sep 20 21:18:39 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn3o8m&sid=BCvXz_PTBp7yLoVYAAAI => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 203/236] ::ffff:192.168.0.111 () {50 vars in 1251 bytes} [Fri Sep 20 21:18:39 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn3o8n&sid=BCvXz_PTBp7yLoVYAAAI => generated 1 bytes in 25019 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 204/237] ::ffff:192.168.0.111 () {56 vars in 1359 bytes} [Fri Sep 20 21:19:04 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn47kf&sid=BCvXz_PTBp7yLoVYAAAI => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 206/238]  () {16 vars in 245 bytes} [Fri Sep 20 21:19:05 2024] GET /api/health/8M4F88S8ooi4sMbLBfkkV7ctWwgibW6V => generated 71 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 206/239] ::ffff:192.168.0.111 () {50 vars in 1251 bytes} [Fri Sep 20 21:19:04 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn47kg&sid=BCvXz_PTBp7yLoVYAAAI => generated 1 bytes in 25003 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 207/240] ::ffff:192.168.0.111 () {56 vars in 1359 bytes} [Fri Sep 20 21:19:29 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn4qvs&sid=BCvXz_PTBp7yLoVYAAAI => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 209/241] ::ffff:192.168.0.111 () {34 vars in 534 bytes} [Fri Sep 20 21:19:52 2024] GET /api/health/8M4F88S8ooi4sMbLBfkkV7ctWwgibW6V => generated 71 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 209/242] ::ffff:192.168.0.111 () {50 vars in 1251 bytes} [Fri Sep 20 21:19:29 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn4qvs&sid=BCvXz_PTBp7yLoVYAAAI => generated 1 bytes in 25002 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 210/243] ::ffff:192.168.0.111 () {56 vars in 1359 bytes} [Fri Sep 20 21:19:54 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn5a6z&sid=BCvXz_PTBp7yLoVYAAAI => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 212/244]  () {16 vars in 245 bytes} [Fri Sep 20 21:20:05 2024] GET /api/health/8M4F88S8ooi4sMbLBfkkV7ctWwgibW6V => generated 71 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 212/245] ::ffff:192.168.0.111 () {50 vars in 1251 bytes} [Fri Sep 20 21:19:54 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn5a6z&sid=BCvXz_PTBp7yLoVYAAAI => generated 1 bytes in 25013 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 214/246] ::ffff:192.168.0.111 () {56 vars in 1359 bytes} [Fri Sep 20 21:20:19 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn5tih&sid=BCvXz_PTBp7yLoVYAAAI => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 214/247] ::ffff:192.168.0.111 () {50 vars in 1251 bytes} [Fri Sep 20 21:20:19 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn5tih&sid=BCvXz_PTBp7yLoVYAAAI => generated 1 bytes in 25018 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 216/248] ::ffff:192.168.0.111 () {56 vars in 1359 bytes} [Fri Sep 20 21:20:45 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn6cum&sid=BCvXz_PTBp7yLoVYAAAI => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 217/249]  () {16 vars in 245 bytes} [Fri Sep 20 21:21:05 2024] GET /api/health/8M4F88S8ooi4sMbLBfkkV7ctWwgibW6V => generated 71 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 217/250] ::ffff:192.168.0.111 () {50 vars in 1250 bytes} [Fri Sep 20 21:20:45 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn6cum&sid=BCvXz_PTBp7yLoVYAAAI => generated 1 bytes in 25004 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 219/251] ::ffff:192.168.0.111 () {56 vars in 1359 bytes} [Fri Sep 20 21:21:10 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn6w68&sid=BCvXz_PTBp7yLoVYAAAI => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 219/252] ::ffff:192.168.0.111 () {50 vars in 1251 bytes} [Fri Sep 20 21:21:10 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn6w68&sid=BCvXz_PTBp7yLoVYAAAI => generated 1 bytes in 25009 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 221/253] ::ffff:192.168.0.111 () {56 vars in 1359 bytes} [Fri Sep 20 21:21:35 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn7fhl&sid=BCvXz_PTBp7yLoVYAAAI => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 221/254] ::ffff:192.168.0.111 () {50 vars in 1251 bytes} [Fri Sep 20 21:21:35 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn7fhl&sid=BCvXz_PTBp7yLoVYAAAI => generated 1 bytes in 25012 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 223/255] ::ffff:192.168.0.111 () {56 vars in 1358 bytes} [Fri Sep 20 21:22:00 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn7yt2&sid=BCvXz_PTBp7yLoVYAAAI => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 224/256]  () {16 vars in 245 bytes} [Fri Sep 20 21:22:05 2024] GET /api/health/8M4F88S8ooi4sMbLBfkkV7ctWwgibW6V => generated 71 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 224/257] ::ffff:192.168.0.111 () {50 vars in 1250 bytes} [Fri Sep 20 21:22:00 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn7yt3&sid=BCvXz_PTBp7yLoVYAAAI => generated 1 bytes in 25003 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 225/258] ::ffff:192.168.0.111 () {56 vars in 1359 bytes} [Fri Sep 20 21:22:25 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn8i47&sid=BCvXz_PTBp7yLoVYAAAI => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 226/259] ::ffff:192.168.0.111 () {50 vars in 1251 bytes} [Fri Sep 20 21:22:25 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn8i48&sid=BCvXz_PTBp7yLoVYAAAI => generated 1 bytes in 25003 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 227/260] ::ffff:192.168.0.111 () {56 vars in 1359 bytes} [Fri Sep 20 21:22:50 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn91fj&sid=BCvXz_PTBp7yLoVYAAAI => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 229/261]  () {16 vars in 245 bytes} [Fri Sep 20 21:23:05 2024] GET /api/health/8M4F88S8ooi4sMbLBfkkV7ctWwgibW6V => generated 71 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 230/262] ::ffff:192.168.0.111 () {34 vars in 534 bytes} [Fri Sep 20 21:23:06 2024] GET /api/health/8M4F88S8ooi4sMbLBfkkV7ctWwgibW6V => generated 71 bytes in 0 msecs (HTTP/1.1 200) 2 headers in 71 bytes (3 switches on core 197)
[pid: 30|app: 0|req: 230/263] ::ffff:192.168.0.111 () {50 vars in 1251 bytes} [Fri Sep 20 21:22:50 2024] GET /socket.io/?EIO=4&transport=polling&t=m1bn91fj&sid=BCvXz_PTBp7yLoVYAAAI => generated 1 bytes in 25007 msecs (HTTP/1.1 200) 2 headers in 100 bytes (3 switches on core 199)
[pid: 30|app: 0|req: 232/264] ::ffff:192.168.0.111 () {56 vars in 1358 bytes} [Fri Sep 20 21:23:15 2024] POST /socket.io/?EIO=4&transport=polling&t=m1bn9kqv&sid=BCvXz_PTBp7yLoVYAAAI => generated 2 bytes in 0 msecs (HTTP/1.1 200) 3 headers in 149 bytes (3 switches on core 197)
```

**Comment by TheSelox** [comment]
I'm experiencing the same issue and just wanted to file a bug report.
Everything worked up until a few days ago.

I think there might have been an app update but I can't remember anymore.

I used to run the AIO beta docker-compose file from [here](https://docs.kitchenowl.org/latest/self-hosting/#__tabbed_1_3) on Unraid.

I already tried switching to the regular docker-compose option and also to the AIO version available in the Unraid app store.

I'm using nginx proxy manager and similar to @DevilsDesigns everything works if I access it through any web browser. It just seems to be the Android app.

**Comment by TomBursch** [comment]
Yes you're on a beta version of the client https://github.com/TomBursch/kitchenowl/releases/tag/beta-v0.6.0 which isn't supported by the latest backend. You can either set the docker tag to `:beta` instead of `:latest` or leave the beta program in the play store https://support.google.com/googleplay/answer/7003180?hl=en 

The beta in AIO just refers to the setup method, not the actual software version. I'll try to make that more clear.

**Comment by TheSelox** [comment]
Ok, leaving the beta, setting the :latest tag, and reinstalling the Android app did not work for me. 
Only after setting the docker tag to :beta the app worked again.
Thanks for your help! 🙇‍♂️

**Comment by TomBursch** [comment]
Strange, but can set the tag back to latest once the next latest version has been released.

**Comment by DevilsDesigns** [comment]
The latest android app is .6 you need to manually install the APK from GitHub. The android app play store won't install the 0.5.2 even after leaving the beta. This fixed it for me.

**Comment by diminou91** [comment]
Hello,
For some reason, this is happening to me as well.
I've just moved my installation from one server to another, everything is good on a web browser, but I can't connect with the app anymore.
I'm in latest beta version both on the server and the app.
Can't figure out what is happening since when trying to connect from the app, it simply throw an "username or password is incorrect" and nothing else happened, can't even see the request on the server's logs... :-(

**Comment by TomBursch** [comment]
@diminou91 I think this is an unrelated error since this issue specifically is about an unsupported app version message when using the current stable version with the beta backend.

Feel free to open a new issue for your error. Just some quick ideas, did you make sure that the url you've put into the app is the correct one? It needs to have the scheme (https). Does the web version work on the same device?

**Comment by diminou91** [comment]
@TomBursch you are right, my bad on the reading of the issue!
I restarted both the containers, again, and after a while it started working. I have absolutely no clue what happened nor what changed but hey... It works, I'll chime in if this change or find out what happened!

Sorry for the noise then 😅
